packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -507,22 +565,20 @@ export default class Meeting extends StatelessSparkPlugin {
    * Scenario D: Joining any other way (sip, pstn, convoId, link just need to specify resourceId)
    */
   join(options = {}) {
+    this.ringing(STOP, false);
```

Comment:
NOTE: I pulled outgoing ringing out of this PR because it was gonna be so hacky, it needs to be done with meeting state machine to handle properly.